src/jwt/macros.rs

```diff
@@ -40,3 +40,86 @@ macro_rules! check_symmetric_key {
         key
     }};
 }
+
+#[macro_export]
+macro_rules! check_asymmetric_key {
+    (
+        key: $key:expr,
+        name: $name:expr,
+        notificator: $notificator:expr,
+        key_kind: $key_kind:ty,
+        key_kind_as_str: $key_kind_as_str:expr
```

Comment:
you don't need it
we already have the `$name:expr` value. it's enough

Reply:
here is an example when `key_kind_as_str` is empty

![image](https://user-images.githubusercontent.com/43034350/230468957-3a4c0881-479a-4250-94b5-bdecd57f24aa.png)